include/wx/msw/setup0.h

```diff
@@ -1432,9 +1444,9 @@
 // Should wxDC provide SetTransformMatrix() and related methods?
 //
 // Default is 1 but can be set to 0 if this functionality is not used. Notice
-// that currently wxMSW, wxGTK3 support this for wxDC and all platforms support
-// this for wxGCDC so setting this to 0 doesn't change much if neither of these
-// is used (although it will still save a few bytes probably).
+// that currently only wxMSW supports this so setting this to 0 doesn't change
+// much for non-MSW platforms (although it will still save a few bytes
+// probably).
```

Comment:
Since 49000defcf and 41a2b3e371 SetTransformMatrix() is supported also by wxGCDC so in practice this feature is not supported only by wxGTK2 (among main ports).